src/test/groovy/io/openliberty/tools/gradle/DevTest.groovy

```diff
@@ -164,6 +164,20 @@ class DevTest extends BaseDevTest {
         assertTrue(verifyLogMessage(10000,  "Tests finished."));
     }
 
+    @Test
+    public void restartServerTest() throws Exception {
+        String TASK_GENERATE = "Task :generateFeatures";
```

Comment:
This string is already declared in [BaseDevTest](https://github.com/OpenLiberty/ci.gradle/blob/f614011d3f0ae005bbaa82b6c7da6612f27e0e6c/src/test/groovy/io/openliberty/tools/gradle/BaseDevTest.groovy#L34)

Reply:
Thanks, I only searched in the current file.